sdk/storage/AzureStorageBlob/Source/Data/BlobTransfer+CoreDataProperties.swift

```diff
@@ -29,7 +29,7 @@ import CoreData
 import Foundation
 
 extension BlobTransfer {
-    @nonobjc internal class func fetchRequest() -> NSFetchRequest<BlobTransfer> {
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<BlobTransfer> {
```

Comment:
Do these (this one, and the two at the end of the file below) need to be made public? Since `BlobTransfer` is part of the public API, these methods will also become part of the public API.

Reply:
I don't think so. I think this may have just been the result of regenerating the NSManagedObject class.

Reply:
Fixed.